### ISSUE TITLE
GPU pilots with fully occupied GPUs should be counted as running

### ISSUE THREAD

**Issue by bbockelm** [glideinWMS/glideinwms#46]
We are hitting issues with "idle" pilots in our glidein pool that are not really idle.

Each pilot is a partitionable slot with 1 GPU, 4 CPUs, and 4GB RAM.  Our jobs all require 1 GPU, 1 CPU, and 1 GB RAM. 
 Accordingly, once we run out of GPU resources, there's only 3 CPUs and 3 GB of RAM left.

It appears the glidein is considered "idle" if there is a partitionable slot with >0 CPU and >2.5GB RAM.  Can we convert this to:

```
Idle = (CPUs > 0 && (!TotalGPUs || GPUs > 0)) && Memory > 2500
```

Thanks.

**Comment by mambelli** [comment]
This has been fixed in https://cdcvs.fnal.gov/redmine/issues/25769
It is under review. Will be probably in v3.7.4, unless is urgent and should delay 3.7.3.

**Comment by mambelli** [comment]
Released in 3.7.4
